### PR TITLE
add resolver version list

### PR DESCRIPTION
**Pull request by Habbie**
@pspacek I wrote down 3.3.0 which is not entirely correct - can you propose (or push) better text?

### CODE REVIEW

**Comment by pspacek** [comment]
@vcunat Please propose amended text for Knot resolver once you are done with testing on our side. Thank you!

**Comment by pspacek** [comment]
@habbie What do you think about this version?

File: README.md [hunk 1]
```diff
@@ -74,8 +74,20 @@ const domainCheckerInit = {
...
+======================
+
+On or around Feb 1st, 2019, major open source resolver vendors will release updates that implement stricter EDNS handling. Specifically, the following versions introduce this change:
+
+* BIND 9.13.3 (development) and 9.14.0 (production)
+* Knot Resolver already implemented stricter EDNS handling in all current versions
+* PowerDNS Recursor 4.2.0
+* Unbound 1.9.0
+
+Also <a href="#supporters">public DNS providers listed below</a> will disable workarounds over a short period of time.
```
[the remark this answers is not in the document]
Comment (vcunat) [reply README.md]: `s/over/within/` ?

[the remark this answers is not in the document]
Comment (pspacek) [reply README.md]: I see! Apparently I was using this incorrectly for years ... Let me try again.

**Comment by pspacek** [comment]
@Habbie Simplified version is ready for review.
